src/modules/layoutUiRules/rulesUtils.js

```diff
@@ -20,6 +21,21 @@ export const hasAxisTooManyItemsByVisType = (
         : false
 }
 
+export const hasDimensionTooManyItemsByVisType = (
+    visType,
+    dimensionId,
+    numberOfItems
+) => {
+    const maxNumberOfItemsPerDimension = getDimensionMaxNumberOfItemsByVisType(
+        visType,
+        dimensionId
+    )
+
+    return maxNumberOfItemsPerDimension
```

Comment:
Here's another ternary with an explicit boolean return, like a previous one above. Again I would personally avoid returning `true` explicitly, as to me that's a sign that something can be refactored.
In this case the logic looks like this:
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:////Users/martinohlson/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip.htm">
<link rel=File-List
href="file:////Users/martinohlson/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_filelist.xml">
<style>
<!--table
	{mso-displayed-decimal-separator:"\,";
	mso-displayed-thousand-separator:" ";}
@page
	{margin:.75in .7in .75in .7in;
	mso-header-margin:.3in;
	mso-footer-margin:.3in;}
tr
	{mso-height-source:auto;}
col
	{mso-width-source:auto;}
br
	{mso-data-placement:same-cell;}
td
	{padding-top:1px;
	padding-right:1px;
	padding-left:1px;
	mso-ignore:padding;
	color:black;
	font-size:12.0pt;
	font-weight:400;
	font-style:normal;
	text-decoration:none;
	font-family:Calibri, sans-serif;
	mso-font-charset:0;
	mso-number-format:General;
	text-align:general;
	vertical-align:bottom;
	border:none;
	mso-background-source:auto;
	mso-pattern:auto;
	mso-protection:locked visible;
	white-space:nowrap;
	mso-rotate:0;}
-->
</style>
</head>

<body link="#0563C1" vlink="#954F72">


maxNumberOfItemsPerDimension | numberOfItems >   maxNumberOfItemsPerDimension | output
-- | -- | --
TRUE | TRUE | TRUE
TRUE | FALSE | FALSE
FALSE | TRUE | FALSE
FALSE | FALSE | FALSE



</body>

</html>

That simply means that it can be written as 
```
return maxNumberOfItemsPerDimension && numberOfItems > maxNumberOfItemsPerDimension
```

The rule just above (`hasAxisTooManyItemsByVisType`) could do with the same change as well

Reply:
Refactored.